common/build-style/cmake.sh

```diff
@@ -113,7 +113,11 @@ do_check() {
 
 	: ${make_check_target:=test}
 
-	${make_cmd} ${make_check_args} ${make_check_target}
+	if [ -z "$disable_parallel_check" ]; then
+		${make_cmd} ${makejobs} ${make_check_args} ${make_check_target}
+	else
+		${make_cmd} ${make_check_args} ${make_check_target}
```

Comment:
This is incorrect. `ninja` will run in parallel by default. I would rather change `makejobs` in `common/xbps-src/libexec/xbps-src-docheck.sh`

Reply:
> This is incorrect. `ninja` will run in parallel by default. I would rather change `makejobs` in `common/xbps-src/libexec/xbps-src-docheck.sh`

Sorry, but I can't find `makejobs` in `common/xbps-src/libexec/xbps-src-docheck.sh`. Where should I define it?

Reply:
It's exported from outside. You can change there, though.

Reply:
> Maybe we don't need `disable_parallel_check` altogether?

Successful multicore compilation with tests broken when running in parallel will likely happen.

> `disable_parallel_build` also implies `disable_parallel_check`, which isn't obvious

Documenting that should be enough. You could also export additional XBPS_MAX_MAKEJOBS or so and read it for check if disable_parallel_build is on if you prefer, just make sure to export that in right places.